src/Init/Lean/Data/Json/Parser.lean

```diff
@@ -0,0 +1,314 @@
+/-
+Copyright (c) 2019 Gabriel Ebner. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+
+Authors: Gabriel Ebner, Marc Huisinga
+-/
+prelude
+import Init.Lean.Data.Json.Basic
+import Init.Control.Except
+
+namespace Lean
+
+inductive Quickparse.Result (α : Type)
+| success {} (pos : String.Iterator) (res : α) : Quickparse.Result
+| error {} (pos : String.Iterator) (err : String) : Quickparse.Result
+
+def Quickparse (α : Type) : Type := String.Iterator → Quickparse.Result α
+
+instance Quickparse.inhabited (α : Type) : Inhabited (Quickparse α) :=
+⟨fun it => Quickparse.Result.error it ""⟩
+
+namespace Quickparse
+
+open Result
+
+partial def skipWs : String.Iterator → String.Iterator | it =>
+if it.hasNext then
+  let c := it.curr;
+  if c = '\u0009' ∨ c = '\u000a' ∨ c = '\u000d' ∨ c = '\u0020' then
```

Comment:
In the new Lean 4 codebase, we have been trying to avoid  `∨` and  `∧`. I know this is the approach used in Lean 3. In Lean 4, we have been using the Boolean operators `||` and  `&&` that are used in most programming languages. 
I am not arguing one is better than the other, but I think it is important to have a uniform style in our codebase.